vassal-app/src/main/java/VASSAL/counters/BasicPiece.java

```diff
@@ -1205,6 +1227,9 @@ public List<String> getPropertyNames() {
     l.add(OLD_MAT_BASIC_NAME);
     l.add(OLD_MAT_OFFSET_X);
     l.add(OLD_MAT_OFFSET_Y);
+    l.add(PIECE_UID);
```

Comment:
Was this PIECE_UID missing? I'm not quite sure why this line is being added here?

Reply:
I think this was due to a conflict.